tests/compiles/tstaticlib2.a

```diff
@@ -0,0 +1 @@
+echo 1234
```

Comment:
this is bad for 2 reasons:
* it'll get overwritten (and make git status not clean) after test is run
* we shouldn't check in .a files (even fake ones)

instead, simply create a temporary file with .a extension and remove it in the test (via `createTempFile`)

Reply:
I see

Reply:
No pollution anymore.